rpcs3/Emu/RSX/Overlays/HomeMenu/overlay_home_menu_main_menu.h

```diff
@@ -2,6 +2,7 @@
 
 #include "overlay_home_menu_page.h"
 #include "overlay_home_menu_settings.h"
+#include "overlay_home_menu_savestate.h"
```

Comment:
Why is this included here?

Reply:
I saw settings.h and thought that would be the best place for savestate.h for the build to compile. else I get
```
3>C:\Users\bryso\Documents\GitHub\rpcs3\rpcs3\Emu\RSX\Overlays\HomeMenu\overlay_home_menu_main_menu.cpp(76,30): error C2065: 'home_menu_savestate': undeclared identifier
3>(compiling source file '/Emu/RSX/Overlays/HomeMenu/overlay_home_menu_main_menu.cpp')
3>C:\Users\bryso\Documents\GitHub\rpcs3\rpcs3\Emu\RSX\Overlays\HomeMenu\overlay_home_menu_main_menu.cpp(76,18): error C2672: 'std::make_shared': no matching overloaded function found
3>(compiling source file '/Emu/RSX/Overlays/HomeMenu/overlay_home_menu_main_menu.cpp')
3>    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.43.34604\include\memory(2959,44):
3>    could be 'std::shared_ptr<_Ty> std::make_shared(const remove_extent<_Ty>::type &)'
3>        C:\Users\bryso\Documents\GitHub\rpcs3\rpcs3\Emu\RSX\Overlays\HomeMenu\overlay_home_menu_main_menu.cpp(76,30):
3>        'std::make_shared': invalid template argument for '_Ty', type expected
3>    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.43.34604\include\memory(2950,44):
3>    or       'std::shared_ptr<_Ty> std::make_shared(void)'
3>        C:\Users\bryso\Documents\GitHub\rpcs3\rpcs3\Emu\RSX\Overlays\HomeMenu\overlay_home_menu_main_menu.cpp(76,30):
3>        'std::make_shared': invalid template argument for '_Ty', type expected
3>    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.43.34604\include\memory(2945,44):
3>    or       'std::shared_ptr<_Ty> std::make_shared(const size_t,const remove_extent<_Ty>::type &)'
3>        C:\Users\bryso\Documents\GitHub\rpcs3\rpcs3\Emu\RSX\Overlays\HomeMenu\overlay_home_menu_main_menu.cpp(76,30):
3>        'std::make_shared': invalid template argument for '_Ty', type expected
3>    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.43.34604\include\memory(2940,44):
3>    or       'std::shared_ptr<_Ty> std::make_shared(const size_t)'
3>        C:\Users\bryso\Documents\GitHub\rpcs3\rpcs3\Emu\RSX\Overlays\HomeMenu\overlay_home_menu_main_menu.cpp(76,30):
3>        'std::make_shared': invalid template argument for '_Ty', type expected
3>    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.43.34604\include\memory(2903,44):
3>    or       'std::shared_ptr<_Ty> std::make_shared(_Types ...)'
3>        C:\Users\bryso\Documents\GitHub\rpcs3\rpcs3\Emu\RSX\Overlays\HomeMenu\overlay_home_menu_main_menu.cpp(76,30):
3>        'std::make_shared': invalid template argument for '_Ty', type expected
3>Done building project "emucore.vcxproj" -- FAILED.
```

Reply:
Obviously you have to include it in the cpp, not the h.
You can probably also move the settings.h

Reply:
Done moved both the settings.h and savestate.h to the .cpp